components/Comment.jsx

```diff
@@ -0,0 +1,51 @@
+import {getUserById} from "../services/API.js";
+import {useEffect, useState} from "react";
+import {formatDate} from "../Utils/Utils.jsx";
+
+export const Comment = ({comment}) => {
+    const {body, comment_id, created_at, author, votes} = comment;
+
+    const [user, setUser] = useState({});
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        getUserById(author)
+            .then(data => {
+                setUser(data.user);
+                setLoading(false);
+            })
+            .catch(error => console.error("Error fetching user:", error));
+    }, [author]);
+
+    return (
+        <div className="flex p-4 border rounded-xl mb-4">
```

Comment:
Try and remove these divs and replace with something more semantic, a div should be a last resort as they are very non-friendly accessibility wise.